### PR TITLE
fix(meetings): remove scheduled meetings and use just in samples

### PR DESCRIPTION
**Pull request by chrisadubois**

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/sample/app.js [hunk 1]
```diff
@@ -8,6 +8,8 @@
 /* eslint-disable no-console */
 /* eslint-disable require-jsdoc */
 
+import '@ciscospark/internal-plugin-calendar';
```
Comment (andrewholsted) [inline packages/node_modules/@webex/plugin-meetings/sample/app.js]: Does this work? I didn't think we could import anything inside of `/sample/` but maybe that was the old way

Comment (chrisadubois) [reply packages/node_modules/@webex/plugin-meetings/sample/app.js]: it seems to work, yeah i tested it

**Comment by chrisadubois** [comment]
does it make sense to do the removal? yeah it will change joining a scheduled meeting because you will have to extract it's sipURi or convoId and join via those methods

File: packages/node_modules/@webex/plugin-meetings/sample/app.js (hunk 1 above)
Comment (andrewholsted) [reply packages/node_modules/@webex/plugin-meetings/sample/app.js]: ok. For consistency, I think we should move it to ciscospark.js where we require other plugins. Isn't that what @arun3528 did for device search?

**Comment by chrisadubois** [comment]
ok done

**Comment by chrisadubois** [comment]
@mattnorris am I not approved to merge PRs into master?

**Comment by mattnorris** [comment]
@chrisadubois you have to submit your changes through gauntlet after receiving approval from a DX core team member (me, Adam, Taymoor, etc.). Since we're merging into master now we need to double-check its impact. After that: 

```
git push gauntlet HEAD:master
```

**Pull request merged**